docs/weave-tutorial/06-buckets.md

```diff
@@ -57,64 +61,61 @@ This basically consists of adding _Copy()_ and _Validate()_ to the objects in `c
 
 ## Dive into Code
 
-Let's define `MarketBucket` that will hold `Market` information and write a function that creates a market. This is a basic `morm/model_bucket` without any secondary index.
+Let's define `UserBucket` that will hold `User` information and write a function that creates an user. This is a basic `morm/model_bucket` without any secondary index.
 
 ```go
-type MarketBucket struct {
+type UserBucket struct {
     morm.ModelBucket
 }
 
-func NewMarketBucket() *MarketBucket {
-    b := morm.NewModelBucket("market", &Market{})
-    return &MarketBucket{
-        ModelBucket: b,
+// NewUserBucket returns a new user bucket
+func NewUserBucket() *UserBucket {
+    return &UserBucket{
+        morm.NewModelBucket("user", &User{}),
     }
 }
 ```
 
-Now create your `OrderBookBucket` with secondary indexes. Secondary indexes enable you to insert and query models with ease. Think of this as a SQL index.
+We will demonstrate secondary index usage with `ArticleBucket.`. Secondary indexes enable you to insert and query models with ease. Think of this as a SQL index.
```

Comment:
`We will demonstrate secondary index usage with `ArticleBucket.`. Secondary indexes enable you to insert and query models with ease. Think of this as an SQL index.
`